hermes/probe/create/create_file_test.go

```diff
@@ -0,0 +1,140 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	metricpb "github.com/google/cloudprober/metrics/proto"
+	probepb "github.com/googleinterns/step224-2020/config/proto"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+	"github.com/googleinterns/step224-2020/hermes/probe/mock"
+	journalpb "github.com/googleinterns/step224-2020/hermes/proto"
```

Comment:
Can you reorganize these? The renamed imports should be at the bottom within their own block (i.e. leave a blank line between the renamed imports and those that haven't been renamed). I think that running `gofmt` should do this for you.

Reply:
reorganised